source/fem-dem/gls_vans.cc

```diff
@@ -229,7 +229,7 @@ GLSVANSSolver<dim>::assembleGLS()
     this->velocity_fem_degree,
```

Comment:
Why are the velocity and pressure mapping class member no used here?

Reply:
I missed it but I fixed it. Thanks